### PR TITLE
Storybook: Add controls to InlineLabel story

### LINKED ISSUE #54173: Add controls to `InlineLabel` story
Add support for [controls](https://storybook.js.org/docs/react/essentials/controls) for the various props in a story.

### PR DESCRIPTION
**Pull request by timagixe**
**What this PR does / why we need it**:
Adds controls to InlineLabel story.

**Which issue(s) this PR fixes**:

Fixes #54173

**Special notes for your reviewer**:

### CODE REVIEW

File: packages/grafana-ui/src/components/Forms/InlineLabel.story.tsx [hunk 1]
```diff
@@ -12,18 +12,54 @@ const meta: ComponentMeta<typeof InlineLabel> = {
...
+      control: 'boolean',
+    },
+    interactive: {
+      control: 'boolean',
+    },
+    as: {
+      table: {
+        disable: true,
+      },
+    },
```
Comment (JoaoSilvaGrafana) [inline packages/grafana-ui/src/components/Forms/InlineLabel.story.tsx:35]: An easy way to do this is is adding this to the same level as `argTypes`
```
    controls: {
      exclude: ['as'],
    },
```

File: packages/grafana-ui/src/components/Forms/InlineLabel.story.tsx [hunk 2]
```diff
@@ -12,18 +12,54 @@ const meta: ComponentMeta<typeof InlineLabel> = {
...
+  interactive: false,
+  as: 'label',
+};
+
+Basic.argTypes = {
+  tooltip: {
+    table: {
+      disable: true,
+    },
+  },
```
Comment (JoaoSilvaGrafana) [inline packages/grafana-ui/src/components/Forms/InlineLabel.story.tsx:55]: We should remove `interactive` as well as this only applies to the tooltip one

File: packages/grafana-ui/src/components/Forms/InlineLabel.story.tsx [hunk 3]
```diff
@@ -12,18 +12,54 @@ const meta: ComponentMeta<typeof InlineLabel> = {
...
+    },
+    width: {
+      control: 'text',
+    },
+    transparent: {
+      control: 'boolean',
+    },
+    interactive: {
+      control: 'boolean',
+    },
```
Comment (JoaoSilvaGrafana) [inline packages/grafana-ui/src/components/Forms/InlineLabel.story.tsx:30]: We don't need to include these as Storybook can guess what the controls should be based on the types of the props 😄

**Review by JoaoSilvaGrafana — COMMENTED** [review]
Great job! 🎉  Left a couple of comments about some of the controls but looks great otherwise

**Comment by timagixe** [comment]
Hi @JoaoSilvaGrafana! Thank you for the detailed feedback, I really appreciate that! I have pushed the recommended changes to this PR.

File: packages/grafana-ui/src/components/Forms/InlineLabel.story.tsx (hunk 1 above)
Comment (timagixe) [reply packages/grafana-ui/src/components/Forms/InlineLabel.story.tsx]: This is awesome, for some reason, I didn't come across it when looking through the storybook docs! Thanks!

**Review by JoaoSilvaGrafana — APPROVED** [review]
Great job! Thanks for contributing! 🙌

**Comment by JoaoSilvaGrafana** [comment]
Hi @timagixe could you merge main into this branch again? I think one of the checks is failing because it's missing some of the changes from main.

**Comment by timagixe** [comment]
@JoaoSilvaGrafana completed.

**Pull request merged**